src/OrbitGl/ThreadTrack.cpp

```diff
@@ -447,15 +419,17 @@ void ThreadTrack::OnTimer(const TimerInfo& timer_info) {
     timer_chain = std::make_shared<orbit_client_data::TimerChain>();
   }
 
-  orbit_client_data::TextBox& text_box = timer_chain->emplace_back(timer_info);
+  // Pass ownership to timer_chain. TODO: Pass timer_info as a value instead of reference order to
```

Comment:
Will create a bug if we agree on this.

Reply:
I would opt for not also tackling this in this PR. The PR is already quite large and we should be mindful about this and split into multiple PRs whenever possible.

Reply:
Created the bug, I'm not sure if I will tackle it soon and I don't want to forget.